### PR TITLE
ledger 3.1.2

### PR DESCRIPTION
**Pull request by deild**
Update to latest upstream release, ledger 3.1.2

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/ledger.rb [hunk 1]
```diff
@@ -19,18 +19,11 @@ class Ledger < Formula
...
-    sha256 "2e652fc4b247b9c7cd482bd07aa57a66fc86597d7a564e6ccf93232700a6c8d8"
-  end
-
   def install
     ENV.cxx11
 
-    # Boost >= 1.67 Python components require a Python version suffix
-    inreplace "CMakeLists.txt", "set(BOOST_PYTHON python)",
-                                "set(BOOST_PYTHON python27)"
+    inreplace "doc/ledger3.texi", "Getting help, ,",
```
Comment (fxcoudert) [inline Formula/ledger.rb]: Why?

Comment (deild) [reply Formula/ledger.rb]: Until we have https://github.com/ledger/ledger/pull/1760, in the next version

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
thanks @deild !

**Pull request closed without merging**